web/lib/HabitatEmojiPicker.js

```diff
@@ -0,0 +1,1407 @@
+const TEMPLATE = document.createElement('template');
+TEMPLATE.innerHTML = `
+<style>
+@import '/lib/emoji/emoji.css';
+
+    :host {
+        padding-left: 50px;
+        color: var(--color-grey);
+    }
+
+    #emoji-select span{
+        padding: 5px;
+        width: auto;
+        line-height: 30px;
+        display: inline-table;
+        text-align: center;
+        cursor: pointer;
+        vertical-align: middle;
+        font-size: 28px;
+        transition: -webkit-transform 60ms ease-out;
+        transition: transform 60ms ease-out;
+        transition: transform 60ms ease-out,-webkit-transform 60ms ease-out;
+        transition-delay: 60ms;
+    }
+    #emoji-select span:hover {
+        transition-delay: 0ms;
+        -webkit-transform: scale(1.4);
+        -ms-transform: scale(1.4);
+        transform: scale(1.4);
+    }
+
+    #emoji-popover {
+        z-index: 5;
+        position: absolute;
+        left: 30px;
+        bottom: 30px;
+        border-radius: 30px 30px 30px 30px;
+        border: 1px solid var(--color-bg-invert);
+        background: var(--color-bg);
+        transition-duration: 200ms;
+        transition-delay: 0ms;
+        transform-style: flat;
+        transform-origin: 50% 50% 0;
+        opacity: 0;
+        transition: all 0.2s linear;
+        visibility: hidden;
+    }
+    #emoji-popover.active{
+        visibility: visible;
+        opacity:1;
+        bottom: 50px;
+    }
+
+    .emoji-container {
+        width: 330px;
+        height: 260px;
+    }
+
+    .emoji-selector {
+        position: absolute;
+        top: 5%;
+        bottom: 5%;
+        overflow-y: scroll;
+        text-align: center;
+    }
+    #picker-button {
+        cursor: pointer;
+    }
+    #picker-button span {
+        background-color: var(--color-bg);
+        width: 1em;
+        height: 1em;
+        position: absolute;
+        transform:translate(-50%, -50%);
+        border: 1px solid var(--color-accent-grey);
+        border-radius: 50%;
+        font-size: 3em;
+    }
+</style>
+
+
+<div id="emoji-popover">
+  <div class="emoji-container" id="emoji-container">
+    <div class="emoji-selector">
+      <div id="emoji-select">
```

Comment:
Can we use the native HTML select element, or is it too hard to style as per requirements?

If we intend to use div for selector components we should add some aria attributes.